### PR TITLE
Adds Brand Style Guide to Readme

### PR DESCRIPTION
**Pull request by ricardobaeta**
Signed-off-by: Ricardo Baeta <ricardo@ricardobaeta.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 ![artwork](files/src-artwork.png)
 
-### The purpose of this repo is to gather Cross-Company's Projects Brand Assets.
+The purpose of this repo is to gather Cross-Company's Projects Branding Assets.
```
Comment (marnovo) [inline README.md]:
```suggestion
All things (open!) design at source{d}: branding, guidelines, UI assets, media & co.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,13 @@
```
Comment (marnovo) [inline README.md]: Change the title too to source{d} Design

File: README.md [hunk 3]
```diff
@@ -2,7 +2,13 @@
 
 ![artwork](files/src-artwork.png)
 
-### The purpose of this repo is to gather Cross-Company's Projects Brand Assets.
+The purpose of this repo is to gather Cross-Company's Projects Branding Assets.
+
+## source{d} Brand Style Guide
+
+Here you find the Primary Guidelines that ground our Brand Assets Usage and overall Branding Rules.
```
Comment (marnovo) [inline README.md]: Stop capitalizing random terms
```suggestion
Here you find the primary guidelines that ground our brand assets usage and overall branding rules.

File: README.md [hunk 4]
```diff
@@ -24,12 +30,12 @@
...
-### [meetups](meetups/README.md)
-### [awesome MLonCode](awesome-mloncode/README.md)
-### [gitbase](gitbase/README.md)
-### [Public Git Archive](pga/README.md)
+- [source{d} Media Kit](media-kit/README.md)
+- [go-git](go-git/README.md)
+- [meetups](meetups/README.md)
+- [awesome MLonCode](awesome-mloncode/README.md)
+- [gitbase](gitbase/README.md)
+- [Public Git Archive](pga/README.md)
```
Comment (marnovo) [inline README.md]: Add the swag assets too later

Comment (ricardobaeta) [reply README.md]: Yes, later on.

File: README.md (hunk 2 above)
Comment (ricardobaeta) [reply README.md]: Done.

**Pull request merged**